docs/markdown/snippets/ccrx_toolchain_support.md

```diff
@@ -0,0 +1,14 @@
+## CC-RX compiler for C and CPP
+
+Cross-compilation is now supported for Renesas RX targets with the CC-RX compiler.
+
+The environment path should be set properly for the CC-RX compiler executables.
+The `-cpu` option will be set appropariately based on the `cpu` option in the cross-file as shown in the snippet below.
```

Comment:
I missed this earlier. We don't do this for any other platform and I'm not sure we should do that here, either. With other toolchains the command line argument to use often does not contain the cpu type directly, but something else entirely (manufacturer codes, defines and the like). For consistency it would probably be better to do the same here. Especially since the sample cross file does have `c_args  = ['--cpu=rx600']` meaning the argument would be given to the compiler twice.